SinuS/SinuS/Views/Authentication/VerifyEmailView.swift

```diff
@@ -10,14 +10,25 @@ import SwiftUI
 struct VerifyEmailView: View {
     var body: some View {
         VStack {
-            Spacer()
+            Text("Please verify your email address by clicking on the link in your email. Then tap the button below.").padding()
 
-            Text("Please verify your email address by clicking on the link in your email")
```

Comment:
Yet to determine how to proceed from here: let user continue anyway, or wait for email is verified? User is already logged in at least.